cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -0,0 +1,143 @@
+| `MATCH (n)-[x:A\|:B\|:C*]-() RETURN n`     | Syntax | Deprecated | Replaced by `MATCH (n)-[x:A\|B\|C*]-() RETURN n`
+| <<user-defined-aggregation-functions, User-defined aggregation functions>> | Functionality | Added |
+| <<query-schema-index-introduction, Composite indexes>> | Index | Added |
+| <<query-constraint-node-key, Node Key>> | Index | Added | Neo4j Enterprise Edition only
+| `CYPHER runtime=compiled` (Compiled runtime) | Functionality | Added | Neo4j Enterprise Edition only
+| <<functions-reverse-list,reverse()>> | Function  | Extended | Now also allows a list as input
+| <<functions-max, max()>>, <<functions-min, min()>> | Function  | Extended | Now also supports aggregation over a set containing both strings and numbers
+|===
+
+[[cypher-deprecations-additions-removals-3.3]]
+=== Version 3.3
+[options="header"]
+|===
+| Feature          | Type | Change | Details
+| `START`     | Clause | Removed | Replaced by the built-in procedures `db.nodeManualIndexSeek`, `db.nodeManualIndexSearch`, `db.relationshipManualIndexSeek`, and `db.relationshipManualIndexSearch`
+| `CYPHER runtime=slotted` (Faster interpreted runtime) | Functionality | Added | Neo4j Enterprise Edition only
+| <<functions-max, max()>>, <<functions-min, min()>> | Function  | Extended | Now also supports aggregation over a set containing lists of strings and numbers
+|===
+
+[[cypher-compatibility]]
+== Compatibility
+
+Older versions of the language can still be accessed if required.
+There are two ways to select which version to use in queries.
+
+. Setting a version for all queries
+You can configure your database with the configuration parameter `cypher.default_language_version`, and enter which version you'd like to use (see <<cypher-versions>>).
+Every Cypher query will use this version, provided the query hasn't explicitly been configured as described in the next item below.
+
+. Setting a version on a query by query basis
+The other way is on a query by query basis.
+By simply putting `CYPHER 2.3` at the beginning, that particular query will be executed with the version of Cypher included in Neo4j 2.3.
+
+Below is an example using the `START` clause to access a legacy index:
+
+[source, cypher]
+----
+CYPHER 2.3
+START n=node:nodes(name = "A")
+RETURN n
+----
+
+
+[[cypher-start-compatibility]]
+=== Accessing entities by id via START
+
+In versions of Cypher prior to 2.2 it was also possible to access specific nodes or relationships using the `START` clause.
+In this case you could use a syntax like the following:
+
+[source, cypher]
+----
+CYPHER 1.9
+START n=node(42)
+RETURN n
+----
+
+[NOTE]
+The use of the `START` clause to find nodes by ID was deprecated from Cypher 2.0 onwards and is now entirely disabled in Cypher 2.2 and up.
+You should instead make use of the `MATCH` clause for starting points.
+See <<query-match>> for more information on the correct syntax for this.
```

Comment:
The link for <<query-match>> doesn't work, but that is expected, right?

Reply:
Correct (only works when it's all built, and the Docs team check and verify that this is so)

Reply:
👍